pkgs/servers/monitoring/mimir/default.nix

```diff
@@ -1,18 +1,25 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub, nixosTests }:
 buildGoModule rec {
   pname = "mimir";
-  version = "2.0.0";
+  version = "2.1.0";
 
   src = fetchFromGitHub {
     rev = "${pname}-${version}";
     owner = "grafana";
     repo = pname;
-    sha256 = "T1lljhC/TS3eoIc9AFo9Oy9/COM/XnfcwtkU618YCdM=";
+    sha256 = "sha256-n7Vzp/GQIC+Mryu9SycMZ3ScPo5O+5tA4TdigpKzmLU=";
   };
 
   vendorSha256 = null;
 
-  subPackages = [ "cmd/mimir" ];
+  subPackages = [
+    "cmd/mimir"
+    "cmd/mimirtool"
```

Comment:
This is a great addition that I missed!

Reply:
Should we add environment.systemPackages in the module to make sure the user has access to the mimirtool ?

Reply:
See https://github.com/NixOS/nixpkgs/pull/175189/commits/373739ea4133e31a4f7963b826fef9d935ec83de